tests/api.c

```diff
@@ -314,6 +314,44 @@ static void test_wolfSSL_CTX_use_PrivateKey_file(void)
 }
 
 
+static void test_wolfSSL_get_privateKey(void)
+{
+#if defined(OPENSSL_EXTRA)
+#if !defined(NO_FILESYSTEM) && !defined(NO_CERTS) && !defined(NO_RSA)
+
+    WOLFSSL_CTX *ctx;
+    WOLFSSL *ssl;
+
+    AssertNotNull(ctx = wolfSSL_CTX_new(wolfSSLv23_server_method()));
+
+    /* test for if key owned */
+    AssertTrue(wolfSSL_CTX_use_certificate_file(ctx, svrCertFile, SSL_FILETYPE_PEM));
+    AssertTrue(wolfSSL_CTX_use_PrivateKey_file(ctx, svrKeyFile, SSL_FILETYPE_PEM));
+    AssertNotNull(ssl = wolfSSL_new(ctx));
+
+    assert(wolfSSL_get_privatekey(ssl) != NULL);
```

Comment:
Have you verified this returns the correct value?  It may be helpful to add a XMEMCMP() or similar to verify that this is working correctly against a known value.